apps/parsley/package.json

```diff
@@ -135,9 +135,9 @@
     "ts-jest": "29.1.2",
     "ts-node": "10.9.1",
     "typescript": "5.1.3",
-    "vite": "4.5.2",
-    "vite-plugin-checker": "0.6.2",
-    "vite-plugin-env-compatible": "1.1.1",
+    "vite": "5.2.11",
+    "vite-plugin-checker": "0.6.4",
+    "vite-plugin-env-compatible": "2.0.1",
     "vite-tsconfig-paths": "4.3.2",
```

Comment:
Is this something we should try to fix?
<img width="904" alt="Screenshot 2024-05-14 at 3 18 49 PM" src="https://github.com/evergreen-ci/ui/assets/47064971/ccdb3691-8c62-4e80-9df1-19b93d0f863c">

Text content:
```
The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
```

Reply:
I looked into this and it's not just a one-line change, so I opened DEVPROD-7396 to fix (I think the CJS build will be removed entirely in Vite 6).